test/asynchronous/helpers.py

```
@@ -404,5 +404,8 @@ def is_alive(self):
 
     async def run(self):
         if self.target:
```

When would "target" ever be None?

---

Good point, we'd want it to throw an error here instead of being a silent no-op.